adaptdl/adaptdl/torch/scaling_rules_test.py

```diff
@@ -199,8 +285,10 @@ def nan_objective(tensor):
     params = torch.autograd.Variable(params_t, requires_grad=True)
     sgd = torch.optim.SGD([params], lr=0.1)
     adp = Mock(require_backward_grad_sync=True)
-    ada = adascale.AdaScale(adp, sgd, accum_scale=1.0, num_replicas=1,
-                            patch_optimizer=True)
+    gns = GradientNoiseScale(adp, sgd, accum_scale=1.0, num_replicas=1)
```

Comment:
I think test_nan() should be moved into a gradient_noise_scale_test.py file, with the AdaScale object removed.